ssl/statem/statem_lib.c

```diff
@@ -2418,7 +2418,6 @@ int ssl_get_min_max_version(const SSL_CONNECTION *s, int *min_version,
 {
     int version, tmp_real_max;
     int hole;
-    const SSL_METHOD *single = NULL;
```

Comment:
`single` is also mentioned in the multi-line comment at (old) lines 2463 and 2466, so that comment should be updated too

Reply:
Removed `single`  from comment